test_psen_scan_v2_prerelease/test/integrationtest_ros_scanner_launch.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+
+import unittest
+import rospy
+
+class TestStartup(unittest.TestCase):
+    def test_startup(self):
+        i = 0
+        expectedTopic = ['/laser_scanner/scan', 'sensor_msgs/LaserScan']
+        while not expectedTopic in rospy.get_published_topics():
+            rospy.sleep(rospy.Duration(0.5))
+            self.assertLess(i, 10, msg="node did not come up in the expected time") # wait 5s max
```

Comment:
I would not rely on a topic being published without attached hardware. We could instead just check if the node comes up: https://github.com/PilzDE/pilz_robots/blob/8c570447a73730b2f5e8538104eb7b6418389472/prbt_hardware_support/test/include/prbt_hardware_support/ros_test_helper.h#L46